docs/guide/exe_sim.rst

```diff
@@ -62,18 +62,6 @@ Summary:
    makes sense for a single-clock-domain module where it's typical to want
    to capture one posedge eval() and one negedge eval().
 
-.. option:: +verilator+prof+threads+file+<filename>
-
-   Deprecated. Alias for :vlopt:`+verilator+prof+exec+file+\<filename\>`
-
-.. option:: +verilator+prof+threads+start+<value>
-
-   Deprecated. Alias for :vlopt:`+verilator+prof+exec+start+\<value\>`
-
-.. option:: +verilator+prof+threads+window+<value>
-
-   Deprecated. Alias for :vlopt:`+verilator+prof+exec+window+\<value\>`
-
```

Comment:
Please have the docs still describe the old options, e.g. "Removed in 4.020. Was an alias..."

Because our web docs remain all-version inclusive.  Also helps those that get bit by these being removed.